htdocs/core/lib/tax.lib.php

```diff
@@ -152,7 +153,7 @@ function tax_by_thirdparty($type, $db, $y, $date_start, $date_end, $modetax, $di
     	$sql.= " ".MAIN_DB_PREFIX."societe as s,";
     	$sql.= " ".MAIN_DB_PREFIX.$invoicedettable." as d" ;
     	$sql.= " LEFT JOIN ".MAIN_DB_PREFIX."product as p on d.fk_product = p.rowid";
-    	$sql.= " WHERE f.entity = " . $conf->entity;
+    	$sql.= " WHERE f.entity IN (".getEntity($invoicetable).")";
```

Comment:
@atm-quentin 
getEntity('invoice') instead please !!!

Reply:
Why use 'invoice' when dolibarr already use 'facture' ? 
https://github.com/Dolibarr/dolibarr/blob/77582b2a4127ca4d90c2fc16b011586ddf5dd751/htdocs/compta/facture/list.php#L402

Reply:
@atm-quentin 
the multicompany module can use 'invoice' and 'facture', it's just to put everything in English

Reply:
using $invoicetable permit to use same field for supplier invoice and customer invoice

Reply:
@atm-quentin 
ok i see